__tests__/utils/fhir/resourceCreation.test.ts

```diff
@@ -438,3 +521,45 @@ describe('createCopiedPatientResource', () => {
     expect(patient.id).not.toEqual(PATIENT_RESOURCE.id);
   });
 });
+
+describe('generateTestCaseMRGroup', () => {
+  test.only('returns a group array with all populations from MeasureReport', () => {
```

Comment:
```suggestion
  it('returns a group array with all populations from MeasureReport', () => {
```
Not a huge deal, but to be consistent with the rest of the file maybe use "it" instead of "test" :)